sddi-base/Dockerfile

```diff
@@ -23,228 +39,193 @@ RUN set -ex && \
   curl -o /wheels/ckanext-hierarchy.txt https://raw.githubusercontent.com/ckan/ckanext-hierarchy/${CKANEXT_HIERARCHY_VERSION}/requirements.txt && \
   ls -lah /wheels
 
-# ckanext-grouphierarchy ######################################################
-ARG CKANEXT_SDDI_VERSION="1.1.4"
-ENV CKANEXT_SDDI_VERSION=${CKANEXT_SDDI_VERSION}
-
-RUN set -ex && \
-  pip wheel --wheel-dir=/wheels \
-    git+https://github.com/tum-gis/ckanext-grouphierarchy-sddi.git@${CKANEXT_SDDI_VERSION}#egg=ckanext-grouphierarchy && \
-  pip wheel --wheel-dir=/wheels -r \
-    https://raw.githubusercontent.com/tum-gis/ckanext-grouphierarchy-sddi/${CKANEXT_SDDI_VERSION}/requirements.txt && \
-  curl -o /wheels/ckanext-grouphierarchy.txt \
-    https://raw.githubusercontent.com/tum-gis/ckanext-grouphierarchy-sddi/${CKANEXT_SDDI_VERSION}/requirements.txt && \
-  ls -lah /wheels
-
-# ckanext-relation ############################################################
-ARG CKANEXT_RELATION_VERSION="1.0.3"
-ENV CKANEXT_RELATION_VERSION=${CKANEXT_RELATION_VERSION}
+# ckanext-envvars
+ENV ENVVARS_GIT_URL=https://github.com/ckan/ckanext-envvars/
+ENV ENVVARS_GIT_BRANCH=v0.0.6
```

Comment:
I suggest we stick a consistent naming for theses variables. `CKANEXT_XXXX_VERSION` is used for the other extensions.

Reply:
The naming has been solved.